src/main/java/com/cs56fitnessapp/models/Meal.java

```diff
@@ -1,29 +1,117 @@
 package com.cs56fitnessapp.models;
+import java.util.ArrayList;
+import java.time.LocalDate;
+
 
 /**
- * @author Nerissa Hsieh
+ * @author Nerissa Hsieh & Jimwell Castillo
  * Date Created: 10/03/17
- * Date Modified: 10/03/17
+ * Date Modified: 10/20/17
  */
 
-public class Meal implements DietFacts {
+public class Meal implements DietFacts{
+    private LocalDate date;
+    private MealTitle title;
+    private ArrayList <FoodEntry> foodEntriesList;
+
+    //    Old Constructor,commented out
+    //    public Meal(LocalDate date, String title, ArrayList <FoodEntry> foodEntriesList, MealTitle title1) {
+    //        this.date = date;
+    //        this.title = title1;
+    //        this.foodEntriesList = new ArrayList<>();
+    //     }
+
+    /**
+     * Edited 10.27.17, Jim
+     * **/
+
+
     @Override
     public int getCaloriesIn() {
-        return 0;
+
+        int totalCalories = 0;
+        for(int i = 0; i<this.foodEntriesList.size(); i++){
+            totalCalories += this.foodEntriesList.get(i).getCalories();
+            }
+        return totalCalories;
     }
 
     @Override
     public double getTotalFatIn() {
-        return 0;
+        int totalFat = 0;
+        for(int i = 0; i<this.foodEntriesList.size(); i++){
```

Comment:
For the methods that return `double` local variable e.g. `totalFat` should be of  `double` type.